### ISSUE TITLE
Mapping 2DIF: Dataset_Release_Place wird nicht gemappt

### ISSUE THREAD

**Issue by jaFHP** [McNamara84/ELMO-Enhanced-Laboratory-Metadata-Optimizer#219]
neuer MDE:
![grafik](https://github.com/user-attachments/assets/f9a4f73a-bb95-4f6a-9bd3-5b2622706fd5)

alter MDE:
![grafik](https://github.com/user-attachments/assets/e1bf62bb-7f5d-487f-bc05-44fbff9c0013)

scheint mir ne Konstante zu sein?

**Comment by McNamara84** [comment]
> scheint mir ne Konstante zu sein?

Ja, und zwar eine, die ich vergessen habe einzubauen, wie mir scheint. 😕

**Comment by Ali-GeoFZ** [comment]
Done!
